### DISCUSSION TITLE
Default file manager as pcmanfm

### DISCUSSION THREAD

**Discussion by j0ker70** [i3/i3#4847] [Q&A]
How can I set pcmanfm as my default file manager? I am using Ubuntu 20.04 and my default file manager is nautilus now(mainly used by firefox when it downloads something).

**Comment by vincentbernat** [comment]
You should be able to use `xdg-mime default pcmanfm.desktop inode/directory`.

**Reply by j0ker70** [reply]
Do I have to logout after running the command?

**Reply by vincentbernat** [reply]
No, it should work right away.

**Reply by tim042849** [reply]
I use the following:
`exec --no-startup-id pcmanfm -d`
In the startup section of my i3 config. It doesn't make pcmanfm the default, but ensures that pcman is available at your fingertips and will fire up in seconds after connecting an external usb file device, for instance. Note: no ampersand needed with i3
